src/workerd/io/actor-cache.h

```diff
@@ -407,11 +407,21 @@ class ActorCache final: public ActorCacheInterface {
     // before the read completes.
     const Value value;
     EntryValueStatus valueStatus;
+
+    // This enum indicates how synchronized this entry is with storage.
+    EntrySyncStatus syncStatus = EntrySyncStatus::NOT_IN_CACHE;
   public:
     auto getValueStatus() const {
       return valueStatus;
     }
 
+    // TODO(now): We have `isDirty()`, which tells me we might also want methods for other states,
```

Comment:
Only if we want to keep separate DIRTY and FLUSHING states. If they're going to merge, I don't see much point in having `isFoo` getters for each possible status.